src/pages/Facility/settings/organizations/components/EditFacilityUserRoleSheet.tsx

```diff
@@ -69,7 +69,7 @@ export default function EditUserRoleSheet({
   });
 
   const { mutate: updateRole } = useMutation({
-    mutationFn: (body: { user: string; role: string }) =>
+    mutationFn: (body: { role: string }) =>
```

Comment:
I don't think you need to use a function here at all, you can just specifuy the mutationFn as `mutate(api, {pathParams})`